src/index.html

```diff
@@ -1,12 +1,330 @@
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="link">гаджеты</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="link">планшеты</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="link">фото</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="items-cards">
+        <div class="cards-container items-cards__cards-container">
+          <div class="product-card" data-qa="card">
+            <img
+              class="product-card__image"
+              src="images/imac.jpeg"
+              alt="imac photo">
+
+            <h1 class="product-card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+
+            <div class="product-card__code">Код товара: 195434</div>
+
+            <div class="product-card__reviews">
+              <div class="stars">
+                <div class="stars__item"></div>
+                <div class="stars__item"></div>
+                <div class="stars__item"></div>
+                <div class="stars__item"></div>
+                <div class="stars__item"></div>
+              </div>
+              <div class="feedbacks">Отзывов: 5</div>
+            </div>
+
+            <div class="price">
+              <div class="price__title">Цена:</div>
+              <div class="price__figure">69 999 грн</div>
+            </div>
+
+            <a href="https://ukr.net"
+              class="button" data-qa="card-hover">Купить
+            </a>
+
+          </div>
+
+          <div class="product-card">
+            <img
+              class="product-card__image"
+              src="images/imac.jpeg"
+              alt="imac photo">
+            <h1 class="product-card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+
+            <div class="product-card__code">Код товара: 195434</div>
+
+            <div class="product-card__reviews">
+              <div class="stars">
+                <div class="stars__item"></div>
+                <div class="stars__item"></div>
+                <div class="stars__item"></div>
+                <div class="stars__item"></div>
+                <div class="stars__item"></div>
+              </div>
+              <div class="feedbacks">Отзывов: 5</div>
+            </div>
+
+            <div class="price">
+              <div class="price__title">Цена:</div>
+              <div class="price__figure">69 999 грн</div>
+            </div>
+
+            <a href="https://ukr.net"
+              data-qa="hover"
+              class="button">Купить
+            </a>
+
+          </div>
+
+          <div class="product-card">
+            <img
+              class="product-card__image"
+              src="images/imac.jpeg"
+              alt="imac photo">
```

Comment:
```suggestion
            <img
              class="product-card__image"
              src="images/imac.jpeg"
              alt="imac photo"
            >
```
Закрывающая скобка на одном уровне с открывающей